src/PrestaShopBundle/Controller/Admin/Configure/AdvancedParameters/ShopController.php

```diff
@@ -42,6 +43,8 @@ class ShopController extends FrameworkBundleAdminController
     /**
      * Search for shops by query.
      *
+     * @AdminSecurity("is_granted('create', request.get('_legacy_controller')) || is_granted('update', request.get('_legacy_controller')) || is_granted('delete', request.get('_legacy_controller')) || is_granted('read', request.get('_legacy_controller'))")
```

Comment:
This route is used by the Multistore header (it was introduced in https://github.com/PrestaShop/PrestaShop/pull/21593). It can be called on any backoffice page, so I think this security is going to break the behavior.

It's a route that can be called by any controller, on any backoffice page so ... I guess it cannot be secured ? :/

Reply:
Hmmm I guess you're right